pyfakefs/fake_filesystem.py

```diff
@@ -247,6 +248,7 @@ def __init__(
         self._add_root_mount_point(total_size)
         self._add_standard_streams()
         self.dev_null: Any = FakeNullFile(self)
+        self._create_temp_dir()
```

Comment:
This is an unexpected behavior change to pyfakefs. I found a lot of unit tests that create a /tmp directory in their fake filesystem object, and they will fail on a new pyfakefs that already creates one in the fake filesystem.